### ISSUE TITLE
When entering a value that doesn't exist...

### ISSUE THREAD

**Issue by joacim-boive** [LeaVerou/awesomplete#9361]
Flow:
Enter: "Javas"
Get: list of entries
Enter "2" ("Javas2")
Get: nothing
Enter: [backspace](leaving "Javas" again)
Get: nothing

Expected: list of entries

Tested in Safari Version 8.0.3 (10600.3.18) OS X 10.10.2

**Comment by LeaVerou** [comment]
This is weird. I cannot reproduce it with my Safari 8.0.3. Pressing backspace and deleting the 2 just works and shows the list again. Can anyone else reproduce?

**Comment by ImBobby** [comment]
I tried and facing same issue. Tested in Version 8.0.3 (10600.3.18) Yosemite. I also made a video just to make it clear. https://www.youtube.com/watch?v=kZAeB8RHJnE

**Comment by brushed** [comment]
Apparently the "hidden" attribute doesn't get toggled when no values match the filter. 
I assume that also the ccs transition doesn't kick in properly when re-entering values that do match.

In the evaluate() function,  I replaced the 

``` javascript
    this.open();
```

by

``` javascript
    this[ this.ul.innerHTML == "" ? "close" : "open" ]();
```

which fixed the issue.

**Comment by LeaVerou** [comment]
I would suggest `this.ul.children.length === 0` as a condition. Pull request please?

**Comment by vlazar** [comment]
Fixed in merged #14804
